includes/pages/user_shifts.php

```diff
@@ -43,18 +43,21 @@ function update_ShiftsFilter_timerange(ShiftsFilter $shiftsFilter, $days) {
   $start_time = $shiftsFilter->getStartTime();
   if ($start_time == null) {
     $start_time = time();
+    if($start_time < 1495238400) {
```

Comment:
would be really nice to somehow construct these timestamps from a date. It seems [PHP is able to do that](http://php.net/manual/en/function.strtotime.php).